frontend/src/mixins/isChangeSelectedWebsite.js

```diff
@@ -0,0 +1,19 @@
+import store from "../store";
+
+export const isChangeSelectedWebsite = {
+    mounted() {
+        store.watch(
+            (state) => state.selectedWebsite,
+            (newValue, oldValue) => {
+                if(newValue !== oldValue) {
+                    this.onWebsiteChange();
+                }
+            }
+        );
+    },
+    methods: {
+        onWebsiteChange: function () {
```

Comment:
No need to define method inside mixin, it should be inside component. And inside watcher you need just check if method exists before you call it